### DISCUSSION TITLE
In Lit 3.0 accessor not working.

### DISCUSSION THREAD

**Discussion by jitendraP-ashutec** [lit/lit#4535] [Q&A]
I have updated My lit library from 2.0 to 3.0. I have used rollup modular bundle library. 

If I a disable  experimentalDecorators then its  giving `Unsupported decorator location:' error. and If I am using it then my library is not working in Angular 14. Its give me below error:

`
  Unexpected keyword or identifier.
   accessor step: boolean; 
`
Do you have any suggestion how to fix it ? what is wrong ? I checked this https://github.com/lit/lit/discussions/4327. its suggest to use bable. Do I need to use babel to fix this issue?

@augustjk 
Thanks
Jitendra Prajapati

**Comment by augustjk** [comment]
Sorry, there's not enough information to help.

Does the source code for the component built with Lit exist in the same project as Angular? If it is separate, is it authored in TypeScript or plain JavaScript? Lit 3 still supports TypeScript experimental decorators (in fact we recommend that) so if you were already using that there should be no change needed.

**Reply by jitendraP-ashutec** [reply]
@augustjk  Sorry, Couldn't able to explain you, do we have any github repo which used rollup version 4.10?

**Reply by augustjk** [reply]
No, it seems our starter repos are on older versions of rollup. Sorry.
